source/guides/administration.rst

```diff
@@ -34,197 +34,131 @@ The basics
-
-* :doc:`Provisioning workflows </onboard/user-provisioning-workflows>` - Learn how to provision and de-provision user accounts.
-* :doc:`Bulk loading data </onboard/bulk-loading-data>` - Import bulk data into Mattermost for teams, channels, users, post content, and more.
-* :doc:`Migration guide </onboard/migrating-to-mattermost>` - Learn how to migrate from other chat services to Mattermost.
-
-Advanced user management configurations
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Advanced user management
+~~~~~~~~~~~~~~~~~~~~~~~~
 
 .. toctree::
     :maxdepth: 1
     :hidden:
 
     AD/LDAP groups </onboard/ad-ldap-groups-synchronization>
+    Use AD/LDAP synchronized groups to manage team or private channel membership </onboard/managing-team-channel-membership-using-ad-ldap-sync-groups>
     Guest accounts </onboard/guest-accounts>
-    Using AD/LDAP synchronized groups to manage team or private channel membership </onboard/managing-team-channel-membership-using-ad-ldap-sync-groups>
     SAML Single Sign-On </onboard/sso-saml>
     SAML Single-Sign-On: technical documentation </onboard/sso-saml-technical>
     SSL client certificate setup </onboard/ssl-client-certificate>
     Certificate-based authentication </onboard/certificate-based-authentication>
 
 * :doc:`AD/LDAP groups </onboard/ad-ldap-groups-synchronization>` - Sync AD/LDAP groups with Mattermost roles and teams.
+* :doc:`Use AD/LDAP synchronized groups to manage team or private channel membership </onboard/managing-team-channel-membership-using-ad-ldap-sync-groups>` - Synchronize your AD/LDAP group with private Mattermost channels and teams.
 * :doc:`Guest accounts </onboard/guest-accounts>` - Create guest accounts to collaborate with individuals outside your organization.
-* :doc:`Using AD/LDAP synchronized groups to manage team or private channel membership </onboard/managing-team-channel-membership-using-ad-ldap-sync-groups>` - Synchronize your AD/LDAP group with private Mattermost channels and teams.
 * :doc:`SAML Single Sign-On </onboard/sso-saml>` - Configure Mattermost to be a SAML 2.0 service provider.
 * :doc:`SAML Single-Sign-On: technical documentation </onboard/sso-saml-technical>` - SAML 2.0 reference documentation for Mattermost.
 * :doc:`SSL client certificate setup </onboard/ssl-client-certificate>` - Configure SSL client certificates for Mattermost Desktop and Web Apps.
 * :doc:`Certificate-Based Authentication </onboard/certificate-based-authentication>` - Set up certificate-based authentication for Mattermost.
 
-Self-hosted server management
+Advanced workspace management
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 .. toctree::
     :maxdepth: 1
     :hidden:
 
-    Managing team and channel members </manage/team-channel-members>
+    Shared channels </onboard/shared-channels>
     Statistics </manage/statistics>
     In-product notices </manage/in-product-notices>
     User satisfaction surveys </manage/user-satisfaction-surveys>
     Health check </manage/health-checks>
```

Comment:
not sure if the health check works for Cloud -- looking at the docs it likely will, but would be good to confirm

Reply:
Confirmed with Maria that health check is supported via Cloud instances. The endpoint is available for cloud instances and administrator could configure their pinging checks for their cloud servers to make sure it's always up and running.